### ISSUE TITLE
Support for multiple ACLs

### ISSUE THREAD

**Issue by mcaulifn** [joyofhex/cookbook-bind#28]
I need to use multiple ACLs for different purposes. I tried to use a list in bind['acl-role'] but that does not work.

**Comment by joyofhex** [comment]
Thanks for your patience. I missed the notification about this issue being created. Indeed the acl-role attribute does not allow multiple entries to be added. However you can directly add multiple roles via adding to the node.default['bind']['acls'] list. It should be an array of hashes with an id (for the name of the ACL) and an array of hosts (which are the ACL entries).

For example in a wrapper recipe:

```ruby
node.default['bind']['acls'] << { 'id' => 'acl-name', 'hosts' => ['! 10.0.1.1', '10/8'] }
```

That should work as you wish. I'm currently working on supporting some custom resources for adding such things as ACLs, and that should be present in the 2.0 version of the cookbook.

**Comment by joyofhex** [comment]
Since writing the preceding comment I've released v2.0.0 of the cookbook. This adds the ability to specify multiple ACLs through use of the bind_acl resource. It is a fairly big change otherwise, but you might want to give it a try to fix this issue.
